positions/junior-software-engineer.md

```diff
@@ -0,0 +1,52 @@
+Junior software engineer
+=================
+
+We're looking for more engineers to join our product team
+and help us revolutionise the design industry.
+
+We have 2 teams - _Demand_ and _Supply_ focusing on
+growth and catalogue/operations of the business respectively.
```

Comment:
We should definitely work further on that section so it's not outdated and it's more motivational.